test/e2e/checkpoint_test.go

```diff
@@ -977,4 +979,164 @@ var _ = Describe("Podman checkpoint", func() {
 		// Remove exported checkpoint
 		os.Remove(fileName)
 	})
+
+	namespaceCombination := []string{
+		"cgroup,ipc,net,uts,pid",
+		"cgroup,ipc,net,uts",
+		"cgroup,ipc,net",
+		"cgroup,ipc",
+		"ipc,net,uts,pid",
+		"ipc,net,uts",
+		"ipc,net",
+		"net,uts,pid",
+		"net,uts",
+		"uts,pid",
+	}
+	for _, share := range namespaceCombination {
+		testName := fmt.Sprintf(
+			"podman checkpoint and restore container out of and into pod (%s)",
+			share,
+		)
+		It(testName, func() {
+			if !criu.CheckForCriu(criu.PodCriuVersion) {
+				Skip("CRIU is missing or too old.")
+			}
+			if !crutils.CRRuntimeSupportsPodCheckpointRestore(podmanTest.OCIRuntime) {
+				Skip("runtime does not support pod restore")
+			}
+			// Create a pod
+			session := podmanTest.Podman([]string{
+				"pod",
+				"create",
+				"--share",
+				share,
+			})
+			session.WaitWithDefaultTimeout()
+			Expect(session.ExitCode()).To(Equal(0))
```

Comment:
This pattern is anti-recommended, please see #10997. Could you fix it (and all instances) to use the recommended form? Thank you!

Reply:
done